README.md

```diff
@@ -134,3 +133,11 @@ If you like this project...
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 > I would like to thank all my coding partners, morning session team and stand up call team for the support.
 <p align="right">(<a href="#acknowledgements">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The MIT link is returning a 404 page, kindly update it 
![image](https://user-images.githubusercontent.com/62705947/217862872-feedfe02-87ee-4a1f-a3f5-b8dab553463d.png)